ckanext/data_qld/templates/header.html

```diff
@@ -59,9 +59,6 @@
 </div>
 {% endblock %}
 <header class="navbar navbar-static-top masthead">
-  {% block header_debug %} {% if g.debug and not g.debug_supress_header %}
-  <div class="debug">Controller : {{ c.controller }}<br/>Action : {{ c.action }}</div>
```

Comment:
why did we remove this?

Reply:
It depends on a Pylons-specific feature and is not really needed anyway.

I could restore it and just call `super()` if you prefer?

Reply:
go with super, we should try to be 'light' touch so that we should keep forward compatibility